extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -79,32 +79,6 @@ class AmpInstagram extends AMP.BaseElement {
   }
 
   /** @override */
-  createPlaceholderCallback() {
-    const placeholder = this.win.document.createElement('div');
-    placeholder.setAttribute('placeholder', '');
-    const image = this.win.document.createElement('amp-img');
-    // This will redirect to the image URL. By experimentation this is
-    // always the same URL that is actually used inside of the embed.
-    image.setAttribute('src', 'https://www.instagram.com/p/' +
```

Comment:
Also it looks like using `api.instagram.com` instead of `www.instagram.com` works similarly though the documentation only mentions the naked domain.